tutorials/using-cloud-vpn-with-checkpoint/index.md

```diff
@@ -0,0 +1,383 @@
+
+# Configuring Route Based IPsec VPN Tunnel
+
+The environment below walks you through an IPSec VPN tunnel setup. Make sure to replace the IP addresses in the sample environment with your own IP addresses.
+
+**Google Cloud Platform**
+
+|Name | Value|                             
+-----|------                                  
+|GCP(external IP)|`35.195.227.26`|
+|VPC CIDR|`10.132.0.0/20`|
+|TUN-INSIDE GCP|`169.254.0.1`|
+|GCP-ASN|`65000`|
+                        
+**Checkpoint**
+
+|Name | Value|
+-----|------
+|Checkpoint Security Gateway(external IP)|`199.203.248.181`|
+|Addresses behind Check Point Security Gateway|`10.0.0.10/24`|
+|TUN-INSIDE- CP|`169.54.0.2`|
+|CP Security Gateway ASN|`65002`|
+
+## Configuring Cloud VPN
+
+### Google Cloud Router UI Configuration
+
+The Google Cloud Router dynamically exchange routes between your Virtual Private Cloud and on-premise networks with Border Gateway Protocol (BGP). For the initial release, Cloud Router supports BGP for Cloud VPN only. Cloud Router works with both legacy networks and sub-networks.
+
+Create a new Cloud router to configure GCP for Site to Site VPN connectivity.
+In Google Cloud Platform Console, select **Networking** > **Cloud Routers** > **Create Router**.
+
+![alt_text](Image_8.PNG)
+
+**Creating a Cloud Router**
+
+|Parameter|Value|Description|
+|---------|-----------|-----|
+|Name|`gcp-to-cp-router`|Name of the cloud router.|
+|Description|           |Description of the cloud router.|
+|Network|`to-cp`|The GCP network the cloud router attaches to. Note: This is the network which manages route information.|
+|Region|`europe-west1`|The home region of the cloud router.Note: Make sure the cloud router is in the same region as the sub-networks it is connecting to.|
+|Google ASN|`65000`|The Autonomous System Number assigned to the cloud router. Use any unused private ASN (64512 - 65534, 4200000000 – 4294967294).|
+
+### VPN Tunnel
+
+In Google Cloud Platform Console, select **Networking** > **Interconnect** > **VPN** to create a VPN connection.
+
+![alt_text](Image_9.PNG)
+
+**Creating a VPN connection**
+
+|Parameter|Value|Description|
+|---------|-----|-----------|
+|Name|`gcp-to-cp-vpn`|Name of the VPN gateway|
+|Description|`VPN tunnel connection between GCP and Check Point Security Gateway`|Description of the VPN connection|
+|Network|`to-cp`|The GCP network the VPN gateway attaches to **Note**: This network will get VPN connectivity|
+|Region|`europe-west1`|The home region of the VPN gateway **Note**: Make sure the VPN gateway is in the same region as the subnetworks it is connecting to.|
+|IP address|`cloud-ip(35.195.227.26`|The static public IP address used by the VPN gateway. An existing, unused, static public IP address within the project can be assigned, or a new one created.|
+
```

Comment:
Add Step 3: Enter the parameters as shown in the following table to create a VPN tunnel: